src/manifest.json

```diff
@@ -20,7 +20,7 @@
     }
   ],
   "share_target": {
-    "action": "/?share-target",
+    "action": "/?utm_medium=PWA&utm_source=share-target&share-target",
```

Comment:
The `utm_source` here is different because it's a different style of launch. In this case it's coming from a share target.